### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by VTY1999**
- Update HTML file, Add popup-window feature
- Update CSS file, Add popup-window feature
- Add Js file, Add popup-window feature
- Add responsive popup for mobile & desktop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,6 +233,31 @@ <h3 class="info" id="info">I'm always interested in hearing about new projects,
         <div class="hr1"></div>
         <div class="hr2"></div>
     </div>
+  </div>
+  <div class="popup">
```
Comment (elyor-doniyorov) [inline index.html:237]: - [ ] Please make sure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). 
Let's compare Figma design and your design below 👇
## Desktop versions:
- Your popup here is not centered. Kindly fix it! 👇 
Screenshot (92)
- Figma design is centered👇 
figma


## Mobile versions:
- Your design here 👇 
Screenshot (90)_LI

- Figma design 👇 
figma

File: main.js [hunk 2]
```diff
@@ -24,4 +24,29 @@ for (let i = 0; i < mobileMenuLinks.length; i += 1) {
...
+seeProject.addEventListener('click', () => {
+  Toogle();
+});
+
+const seeProjectLinks = document.querySelectorAll('.closePopUp');
+for (let i = 0; i < seeProjectLinks.length; i += 1) {
+  seeProjectLinks[i].addEventListener('click', () => {
+    closePopUp();
+  });
+}
```
Comment (elyor-doniyorov) [inline main.js:62]: - [ ] The project requirements say that you should use a JavaScript array of objects to store the information for each project. Kindly use the example below 👇 
Screenshot (91)

- [ ] Also make sure the modal popup data is populated with the data from the projects array

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @VTY1999, 

Good job so far!

To highlight 👏 
- The JavaScript and HTML&CSS linters are set correctly project repository. ✔️ 
- When the user clicks (or taps) the close (X) button, the popup disappears. ✔️ 
- The popup is responsive, it fits mobile and desktop sizes properly. ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉 

### Hi @team ✋🏻, 

Good job implementing changes requested by the previous reviewer 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**